projects/cm_mcu/clocksynth.h

```diff
@@ -29,7 +29,7 @@ int load_clock();
 #define CLOCK_EXPANDER_CHOOSE_CLOCKSYNTH_4XCVR        0x0f
 #define CLOCK_EXPANDER_ENABLE_CLOCKSYNTH              0x13
 #define CLOCK_EXPANDER_RESET_CLOCKSYNTH               0x1b
-#define CLOCK_EXPANDER_CONFIGURATION_PORT_0                                                        \
-  0x06 // configuration port to set I/O port as input or output
+#define CLOCK_EXPANDER_CONFIGURATION_PORT_0           0x06 // configuration port to set I/O port as input or output
+#define CHANGEPAGE_REG_ADDR                           0x01
```

Comment:
since this is now in a header file maybe you should add a CLOCK_ to the start of this preprocessor directive, to make its scope clear and in keeping with other preprocessor directives in this file.